src/test/java/org/broadinstitute/hellbender/utils/nio/GcsNioIntegrationTest.java

```diff
@@ -83,6 +84,28 @@ public void openPrivateFileUsingDefaultCredentials() throws IOException {
         }
     }
 
+    @Test(groups = {"bucket"})
+    public void pathLineIterator() throws IOException {
```

Comment:
Test case should be moved to `PathLineIteratorUnitTest` and renamed to `testPathLineIteratorWithGCS`

Reply:
Done.